scripts/srt-dev.lua

```diff
@@ -0,0 +1,887 @@
+fields.snd_tsbpb_delay = ProtoField.uint16("srt_dev.snd_tsbpb_delay", "Send TsbPd Delay", base.DEC)
+
+-- V adn PT status flag
+local V_state_select = {
+	[1] = "Initial version"
+}
+fields.V_state = ProtoField.uint8("srt_dev.V_state", "V", base.HEX, V_state_select, 0x70)
+local PT_state_select = {
+	[0] = "Reserved",
+	[1] = "MSmsg",
+	[2] = "KMmsg",
+	[7] = "Reserved to discriminate MPEG-TS packet(0x47=sync byte)"
+}
+fields.PT_state = ProtoField.uint8("srt_dev.PT_state", "PT", base.HEX, state_table, 0xF)
+fields.sign = ProtoField.uint16("srt_dev.sign", "Signature", base.HEX)
+local resv_select = {
+	[0] = "Reserved for flag extension or other usage"
+}
+fields.resv = ProtoField.uint8("srt_dev.resv", "Resv", base.DEC, state_table, 0xFC)
+fields.ext_KK_state = ProtoField.uint8("srt_dev.ext_KK_state", "KK_state", base.HEX, KK_state_select, 0x3)
+fields.KEKI = ProtoField.uint32("srt_dev.KEKI", "KEKI", base.DEC)
+fields.cipher = ProtoField.uint8("srt_dev.cipher", "Cipher", base.DEC)
+fields.auth = ProtoField.uint8("srt_dev.auth", "auth", base.DEC)
+fields.SE = ProtoField.uint8("srt_dev.SE", "SE", base.DEC)
+fields.resv1 = ProtoField.uint8("srt_dev.resv1", "resv1", base.DEC)
+fields.resv2 = ProtoField.uint16("srt_dev.resv2", "resv2", base.DEC)
+fields.slen = ProtoField.uint8("srt_dev.slen", "Salt length(bytes)/4", base.DEC)
+fields.klen = ProtoField.uint8("srt_dev.klen", "SEK length(bytes)/4", base.DEC)
+fields.salt = ProtoField.uint32("srt_dev.salt", "Salt key", base.DEC)
+fields.wrap = ProtoField.none("srt_dev.wrap", "Wrap key(s)", base.NONE)
+
+-- Handshake packet, ext_type == SRT_CMD_SMOOTHER filed
+fields.smoother_block = ProtoField.none("srt_dev.smoother_block", "Smoother Block", base.NONE)
+
+-- Handshake packet, ext_type == SRT_CMD_SID filed
+fields.sid_block = ProtoField.none("srt_dev.sid_block", "SID Block", base.NONE)
+
+-- SRT flags
+local flag_state_select = {
+	[0] = "Not set",
+	[1] = "Set"
+}
+fields.srt_opt_tsbpdsnd = ProtoField.uint32("srt_dev.srt_opt_tsbpdsnd", "SRT_OPT_TSBPDSND", base.HEX, flag_state_select, 0x1)
+fields.srt_opt_tsbpdrcv = ProtoField.uint32("srt_dev.srt_opt_tsbpdrcv", "SRT_OPT_TSBPDRCV", base.HEX, flag_state_select, 0x2)
+fields.srt_opt_haicrypt = ProtoField.uint32("srt_dev.srt_opt_haicrypt", "SRT_OPT_HAICRYPT", base.HEX, flag_state_select, 0x4)
+fields.srt_opt_tlpktdrop = ProtoField.uint32("srt_dev.srt_opt_tlpktdrop", "SRT_OPT_TLPKTDROP", base.HEX, flag_state_select, 0x8)
+fields.srt_opt_nakreport = ProtoField.uint32("srt_dev.srt_opt_nakreport", "SRT_OPT_NAKREPORT", base.HEX, flag_state_select, 0x10)
+fields.srt_opt_rexmitflg = ProtoField.uint32("srt_dev.srt_opt_rexmitflg", "SRT_OPT_REXMITFLG", base.HEX, flag_state_select, 0x20)
+fields.srt_opt_stream = ProtoField.uint32("srt_dev.srt_opt_stream", "SRT_OPT_STREAM", base.HEX, flag_state_select, 0x40)
+
+-- ACK fileds
+fields.last_ack_pack = ProtoField.uint32("srt_dev.last_ack_pack", "Last ACK Packet Sequence Number", base.DEC)
```

Comment:
It's better to say "ACK sequence". It has its own definition in the protocol and it sounds rather like "the sequence of the last contiguous received packet increased by 1", there's no need to be so elaborate.